omero/developers/Server/Clustering.rst

```diff
@@ -46,3 +46,50 @@ newly created server will not be used until sessions are manually
 redirected to it.
 
 .. seealso:: :doc:`/developers/Server/ScalingOmero`
+
+Read-Only
+---------
+
+A read-only server offers users the ability to do little more than log
+in and retrieve data. Prohibitions for users of read-only servers
+include that they may not import data or run scripts. Two properties
+control read-only configuration: :property:`omero.cluster.read_only.db`
+for the database and :property:`omero.cluster.read_only.repo` for the
+binary repository.
```

Comment:
In what situation would these be different?

Reply:
I don't know really, I don't much understand the envisaged deployments. But to give an idea of what I'd expect of the difference:

* with only DB being read-only, Bio-Formats probably doesn't need special configuration, it can keep on creating memo files in the main repo
* with only FS being read-only, users can probably still do things like tag images and draw ROIs.